### PR TITLE
Fix some ZFA mapping errors

### LINKED ISSUE #3353: ZFA/Uberon mapping errors
Two of the root unsatisfiable classes found in the merge product of Uberon and ZFA are caused by what are likely simple “mapping errors“, where a ZFA term has been mapped to the wrong Uberon term.

## 'griseum centrale'

'[griseum centrale](http://purl.obolibrary.org/obo/ZFA_0000518)' in ZFA is [part_of](http://purl.obolibrary.org/obo/BFO_0000050) the '[medulla oblongata](http://purl.obolibrary.org/obo/ZFA_0000545)', itself [part_of](http://purl.obolibrary.org/obo/BFO_0000050) the [hindbrain](http://purl.obolibrary.org/obo/ZFA_0000029).

But it is mapped to Uberon’s '[central gray substance of midbrain](http://purl.obolibrary.org/obo/UBERON_0003040)', which is part of, well, the [midbrain](http://purl.obolibrary.org/obo/UBERON_0001891) (which in Uberon is spatially disjoint with the [hindbrain](http://purl.obolibrary.org/obo/UBERON_0002028), hence the unsatisfiability of '[griseum centrale](http://purl.obolibrary.org/obo/ZFA_0000518)').

'[griseum centrale](http://purl.obolibrary.org/obo/ZFA_0000518)' should probably be mapped instead to '[central gray substance of medulla](http://purl.obolibrary.org/obo/UBERON_0002867)', which is [part_of](http://purl.obolibrary.org/obo/BFO_0000050) the '[medulla oblongata](http://purl.obolibrary.org/obo/UBERON_0001896)'.

## coelom

[coelom](http://purl.obolibrary.org/obo/ZFA_0001438) in ZFA is a '[multi-tissue structure](http://purl.obolibrary.org/obo/ZFA_0001488)', and therefore a material entity (though the text definition says it is a “anatomical space”, confusingly enough).

It is mapped to Uberon’s '[coelemic cavity lumen](http://purl.obolibrary.org/obo/UBERON_0002323)', which is an '[anatomical cavity](http://purl.obolibrary.org/obo/UBERON_0002553)' and ultimately an immaterial anatomical entity.

It should be mapped instead to Uberon’s [coelom](http://purl.obolibrary.org/obo/UBERON_0011997), which is also defined as '[multi-tissue structure](http://purl.obolibrary.org/obo/ZFA_0001488)' and therefore a material entity (the term refers to “the aggregate of the coelemic cavity lumen **plus the membranes that line the lumen**”).

### LINKED ISSUE #3356: ZFA/Uberon issues: inferior raphe nucleus
In ZFA, '[inferior raphe nucleus](http://purl.obolibrary.org/obo/ZFA_0000366)' is a '[hindbrain nucleus](http://purl.obolibrary.org/obo/UBERON_0009662)' and therefore [part_of](http://purl.obolibrary.org/obo/BFO_0000050) the [hindbrain](http://purl.obolibrary.org/obo/ZFA_0000029).

The ZFA term is mapped to Uberon’s '[dorsal raphe nucleus](http://purl.obolibrary.org/obo/UBERON_0002043)', which is ultimately [part_of](http://purl.obolibrary.org/obo/BFO_0000050) the [midbrain](http://purl.obolibrary.org/obo/UBERON_0001891) (through several independent relationships).

This is a direct contradiction, and a violation of the disjointness between [hindbrain](http://purl.obolibrary.org/obo/UBERON_0002028) and [midbrain](http://purl.obolibrary.org/obo/UBERON_0001891) in Uberon.

I believe this may be a mapping error. According to [Beecher, Beamer & Bartlett 2019](https://www.sciencedirect.com/science/article/pii/B9780128000502000061), the “raphe nuclei” are

* the _superior raphe nuclei_
  * the _dorsal raphe nucleus_
  * the _median raphe nucleus_
  * the _supralemniscal nucleus_
* the _inferior raphe nuclei_
  * the _nucleus raphe obscurus_
  * the _nucleus raphe pallidus_
  * the _nucleus raphe magnus_

So it seems dubious that ZFA’s '[inferior raphe nucleus](http://purl.obolibrary.org/obo/ZFA_0000366)' should be mapped to '[dorsal raphe nucleus](http://purl.obolibrary.org/obo/UBERON_0002043)' which is (according to the classification above) a **superior** raphe nucleus.

In Uberon, '[nucleus raphe obscurus](http://purl.obolibrary.org/obo/UBERON_0002684)', '[nucleus raphe pallidus](http://purl.obolibrary.org/obo/UBERON_0002157)', and '[nucleus raphe magnus](http://purl.obolibrary.org/obo/UBERON_0002156)' are the three parts making up the '[medullay raphe nuclear complex](http://purl.obolibrary.org/obo/UBERON_0002692)'. So assuming that what ZFA calls the _inferior raphe nucleus_ is the set of three nuclei that Beecher, Beamer & Bartlett call the _inferior raphe nuclei_, the ZFA term should be mapped to '[medullay raphe nuclear complex](http://purl.obolibrary.org/obo/UBERON_0002692)' instead.

Of note, if the above is true, that is, if _inferior raphe nucleus_ is not the same thing as _dorsal raphe nucleus_, then there is an also a similar error in GO: '[inferior raphe nucleus development](http://purl.obolibrary.org/obo/GO_0021724)' is said to '[results in development of](http://purl.obolibrary.org/obo/RO_0002296) the '[dorsal raphe nucleus](http://purl.obolibrary.org/obo/UBERON_0002043)'.

### PR DESCRIPTION
**Pull request by gouttegd**
This PR fixes a handful of “simple” mapping errors¹ between ZFA and Uberon:

* ZFA’s '[griseum centrale](http://purl.obolibrary.org/obo/ZFA_0000518)' is remapped to Uberon’s '[central gray substance of medulla](http://purl.obolibrary.org/obo/UBERON_0002867)' (#3353);
* ZFA’s [coelom](http://purl.obolibrary.org/obo/ZFA_0001438) is remapped to Uberon’s [coelom](http://purl.obolibrary.org/obo/UBERON_0011997) (#3353);
* ZFA’s '[inferior raphe nucleus](http://purl.obolibrary.org/obo/ZFA_0000366)' is remapped to Uberon’s '[medullary raphe nuclear complex](http://purl.obolibrary.org/obo/UBERON_0002692)' (#3356).

See the linked issues (or the individual commit messages) for the rationale behind each remapping.

closes #3353 
closes #3356

---
¹ What I mean by “simple mapping error”: the error is simply that a ZFA term is mapped to the wrong Uberon term, but none of the term on either side is otherwise incorrect.

### CODE REVIEW

**Comment by gouttegd** [comment]
PR updated to remove entirely the mapping with ZFA’s '[inferior raphe nucleus](http://purl.obolibrary.org/obo/ZFA_0000366)', as suggested in the comments of #3356.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -61074,6 +61071,7 @@ xref: FMA:72595
 xref: HBA:265504914
 xref: neuronames:756 {source="BIRNLEX:2638"}
 xref: UMLS:C0262207 {source="BIRNLEX:2638"}
+xref: ZFA:0000518
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:61074]: Could it be that ZFA is not correct? [Olson et al.,](https://pmc.ncbi.nlm.nih.gov/articles/PMC6084820/) show that the 'Griseum centrale' is an homologue of the 'periaqueductal gray', which is found in the midbrain (and it is a synonym of 'central gray substance of midbrain' UBERON:0003040). This is also stated by [Blasiak et al](https://www.frontiersin.org/journals/molecular-neuroscience/articles/10.3389/fnmol.2022.984524/full). Therefore, I think that the maping is correct, but the ZFA textual and logical definition are not.

**Review by aleixpuigb — CHANGES_REQUESTED** [review]
I agree with the changes in 'coelom' and the 'inferior raphe nucleus', but I think there might be a mistake in the ZFA for 'Griseum centrale'.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo]: > I think at this point, regardless of where the 'griseum centrale' really is in Zebrafish (hindbrain, as ZFA says, or midbrain?), the only sensible action for Uberon is to remove the mapping entirely. Better not to assert anything than to assert something dubious.

I agree, the literature is too conflictive in this case.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**